src/util.ts

```diff
@@ -20,28 +20,53 @@
  * @param {*} state
  */
 export const trackComments = (node, state) => {
-  if (node.leadingComments) {
-    for (const comment of node.leadingComments) {
+  let leadingNode = node,
+    trailingNode = node;
+
+  if (Array.isArray(node)) {
+    leadingNode = node[0];
+    trailingNode = node[node.length - 1];
+  }
```

Comment:
style nit: this could be simplified to:
```
const [leadingNode, trailingNode] = Array.isArray(node)
  ? [node[0], node[node.length - 1]]
  : [node, node];
```

Reply:
Monzo generally dislikes the ternary operator so this was much more that influence than anything! I'm ambivalent so happy to change it.

Reply:
At KA the preference is to use `const` as much as possible which leads to more ternary operators in the code.